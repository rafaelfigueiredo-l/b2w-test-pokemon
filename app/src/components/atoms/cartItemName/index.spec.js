import CartItemName from "./";

describe("<CartItemName />", () => {
  const component = <CartItemName>Texto</CartItemName>;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly CartItemName component", () => {
    expect(wrapper.find(".cartItemName")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly CartItemName component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should have a children", () => {
    expect(wrapper.find(".cartItemName").children()).toHaveLength(1);
  });
});
