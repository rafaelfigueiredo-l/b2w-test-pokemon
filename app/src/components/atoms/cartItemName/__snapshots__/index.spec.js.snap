// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartItemName /> should mount correctly CartItemName component 1`] = `
<styled.span
  className="cartItemName"
>
  <span
    className="sc-AxjAm StDqN cartItemName"
  >
    Texto
  </span>
</styled.span>
`;
